content/en/docs/setup/upgrade/notice/index.md

```diff
@@ -1,44 +1,73 @@
 ---
-title: 1.2 Upgrade Notice
-description: Important changes operators must understand before upgrading to Istio 1.2.
+title: 1.3 Upgrade Notice
+description: Important changes to consider when upgrading to Istio 1.3.
 weight: 5
 aliases:
     - /docs/setup/kubernetes/upgrade/notice/
 ---
 
 This page describes changes you need to be aware of when upgrading from
-Istio 1.1 to 1.2.  Here, we detail cases where we intentionally broke backwards
+Istio 1.2 to 1.3.  Here, we detail cases where we intentionally broke backwards
 compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
-familiar with the use and operation of Istio 1.1.
+familiar with the use and operation of Istio 1.2.
 
 For an overview of new features introduced with Istio 1.2, please refer
-to the [1.2 release notes](/about/notes/1.2/).
-
-## Installation and Upgrade
-
-{{< tip >}}
-The configuration model for Mixer has been simplified. Support for
-adapter-specific and template-specific Custom Resources has been
-removed by default in 1.2 and will be removed entirely in 1.3.
-Please move to the new configuration model.
-{{< /tip >}}
-
-Most Mixer CRDs were removed from the system to simplify the configuration
-model, improve performance of Mixer when used with Kubernetes, and improve
-reliability in a variety of Kubernetes environments.
-
-The following CRDs remain:
-
-| Custom Resource Definition name | Purpose |
-| --- | --- |
-| `adapter`| Specification of Istio extension declarations |
-| `attributemanifest` | Specification of Istio extension declarations |
-| `template` | Specification of Istio extension declarations |
-| `handler` | Specification of extension invocations |
-| `rule` | Specification of extension invocations |
-| `instance` | Specification of extension invocations |
-
-In the event you are using the removed mixer configuration schemas, set
-the following Helm flags during upgrade of the main Helm chart:
-`--set mixer.templates.useTemplateCRDs=true --set mixer.adapters.useAdapterCRDs=true`
+to the [1.3 release notes](/about/notes/1.3/).
+
+## Installation and upgrade
+
+We simplified the configuration model for Mixer and removed support for
+adapter-specific and template-specific Custom Resource Definitions (CRDs)
+entirely in 1.3. Please move to the new configuration model.
+
+We removed the Mixer CRDs from the system to simplify the configuration
+model, improve Mixer's performance in Kubernetes deployments, and improve
+reliability in various Kubernetes environments.
+
+## Traffic management
+
+Istio now captures all ports by default. If your services listen on `127.0.0.1`,
+you probably don't specify container ports due to the following reasons:
+
+- To avoid exposing your service.
+- To intentionally bypass Envoy.
+
+If that is the case, you must opt out of port capturing with the
+`traffic.sidecar.istio.io/excludeInboundPorts` option.
+
+{{< warning >}}
```

Comment:
many applications bind to the wildcard `0.0.0.0` which means they are binding to localhost.  Some operators have no idea their applications are wildcard binding.

This warning leaves me scratching my head "Ok so no protocol sniffing for 90% of the applications that wildcard bind by default.  Some applications have no way *NOT* to wildcard bind".  I'd be a little more explicit in this fact up front to help clarify this.  The fact that Envoy connects over `localhost` is really irrelevant... :)

Reply:
1. This feature is completely different than protocol sniffing, it is about capturing all inbound traffic by default. Before only container Ports were captured.
2. Envoy connecting over localhost is not really irrelevant. If you bind to 127.0.0.1 and some other pod calls you, the connection is refused. Now you add Istio and all the sudden the calls work.

I really want to avoid someone accidentally exposing a sensitive endpoint so if it's not clear we should refine this

Reply:
Ok, I'm struggling because the underlaying problem is the behavior. The current text at least warns folks they should be aware of the possibility that their endpoints could be exposed and how to address it. I'd suggest we keep the warning but maybe we can use some better wording. @nmittler Can you help us find the right way to phrase this?

Reply:
I chatted with @howardjohn, who has more background on the changes.  His suggested text:

```
Istio now captures all ports by default. If your don't specify container ports to intentionally
bypass Envoy, you must opt out of port capturing with the 
`traffic.sidecar.istio.io/excludeInboundPorts` option.

Warning: This change exposes any applications listening on `127.0.0.1` since Envoy 
connects over localhost. Opt out to avoid exposing such applications.
```